### ISSUE TITLE
Move AbstractGPs related test from GPLikelihoods here

### ISSUE THREAD

**Issue by theogf** [JuliaGaussianProcesses/AbstractGPs.jl#255]
I removed the tests using AbstactGPs from GPLikelihoods and saved them in [this gist]. They should probably be added here (or ApproximateGPs) to check compatibility.

**Comment by willtebbutt** [comment]
I _think_ I would prefer ApproximateGPs, but just because we already have this as a dependency there. I could be convinced otherwise.

**Comment by theogf** [comment]
I really don't mind either, but somehow I think these tests could still fit `GPLikelihoods` just not in the interface module.

**Comment by willtebbutt** [comment]
Sorry, I'm not sure what you mean. Could you elaborate?

**Comment by theogf** [comment]
Well the whole point of `GPLikelihoods` is to work together with the rest of the ecosystem. So I think we should know when changes in GPL might be incompatible with AppGP or AbsGP.

**Comment by willtebbutt** [comment]
Fair point. Taking another look at the code, it looks like it would be fine in this package (I was worried we'd need some of the code in ApproximateGPs in order to run decent tests, but the ones that we currently have seem fine without it).

Where were you thinking to locate these tests? `test` or `src`?

**Comment by theogf** [comment]
Definitely `test` right now `GPLikelihoods` is not a dependency and that would not make sense to add it.

**Comment by willtebbutt** [comment]
Cool. I'm very happy to review if you open a PR.

**Comment by st--** [comment]
What's the reason for not just having this within the GPLikelihoods package ?

**Comment by theogf** [comment]
You mean in the tests of GPLikelihoods?
